login/models.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib.auth.base_user import AbstractBaseUser
+from django.contrib.auth.models import PermissionsMixin, UserManager
+from django.db import models
+
+
+# KTODO copy role and group models
+
+# Create your models here.
+
```

Comment:
because we are using a custom user model, we need to define a user manager (borrowing from: https://docs.djangoproject.com/en/4.0/topics/auth/customizing/#a-full-example)

```python
class CofkUserManager(BaseUserManager):
    def create_user(self, email, password=None):
        """
        Creates and saves a User with the given email and password.
        """
        if not email:
            raise ValueError('Users must have an email address')

        user = self.model(
            email=self.normalize_email(email),
        )

        user.set_password(password)
        user.save(using=self._db)
        return user

    def create_superuser(self, email, password=None):
        """
        Creates and saves a superuser with the given email and password.
        """
        user = self.create_user(
            email,
            password=password,
        )
        user.is_admin = True
        user.save(using=self._db)
        return user
```